pkgs/by-name/ou/outfieldr/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  fetchFromGitLab,
+  stdenv,
+  zig_0_14,
+}:
+
+let
+  zig = zig_0_14;
+in
+stdenv.mkDerivation rec {
```

Comment:
Maybe consider using `(finalAttrs:` instead of `rec` to improve flexibility on override (see #315337)

Reply:
Ah I saw other packages using that but I understand `rec` better so I went with that. I didn't know there's an advantage to using finalAttrs, I'll change it later today. Thanks 😊

Reply:
Fixed :)